index.js

```diff
@@ -59,16 +60,17 @@
 //. end we list all unique error messages.
 //.
 //. ```js
-//. const Future = require('fluture');
-//. const {retry, exponentially} = require('fluture-retry');
+//. const Future = require ('fluture');
+//. const {retry, exponentially} = require ('fluture-retry');
 //.
 //. //    retriedTask :: Future (Array Error) String
-//. const retriedTask = retry(exponentially(64), 32, task);
+//. const retriedTask = retry (exponentially (64), 32, task);
 //.
-//. retriedTask.fork(
-//.   errors => console.error(
+//. retriedTask.fork (
+//.   errors => console.error (
 //.     `All tries failed. The following errors were encountered: \n  ${
-//.       Array.from(new Set(errors.map(({message}) => message))).join('\n  ')
+//.       Array.from (new Set (errors.map (({message}) => message)))
+//.       .join ('\n  ')
```

Comment:
> Any idea what might be going on here?

This line begins with a `.` which indicates to doctest that this line continues the previous line. For now, I suggest restructuring this paragraph to avoid beginning a line with a `.`. If you feel so inclined, please open a doctest issue for this. At the very least we should provide a meaningful error message in such cases, but ideally we would ignore a line with a leading `.` which follows an “insignificant” line.